django/applications/catmaid/control/cropping.py

```diff
@@ -328,11 +306,16 @@ def extract_substack(job) -> List:
         crop_y_max_px = to_y_index(crop_y_max, job.ref_stack, job.zoom_level, False)
         crop_width_px = crop_x_max_px - crop_x_min_px
         crop_height_px = crop_y_max_px - crop_y_min_px
-        # Crop all images (Geometry: width, height, xOffset, yOffset)
-        crop_geometry = Geometry(crop_width_px, crop_height_px,
-            crop_x_min_px, crop_y_min_px)
+
+        # Crop all images (left, upper, right, lower)
+        crop_geometry = (crop_x_min_px, crop_y_min_px, crop_x_min_px + crop_width_px, crop_y_min_px + crop_height_px)
+        out_stack = []
         for img in cropped_stack:
-            img.crop(crop_geometry)
+            cropped = img.crop(crop_geometry)
+            cropped.load()
```

Comment:
Again, not sure about the necessity of this `load`: if it weren't necessary, we could use a list comprehension as with the rotations.

Reply:
Yes, it can be remove here too (image date is created) and I agree that we should.

Reply:
Since I merged this branch already, I applied this change and removed `load()` here, too.